vms/platformvm/txs/executor/camino_tx_executor.go

```diff
@@ -2364,3 +2365,25 @@ func outputsAreEqual(outs1, outs2 []*avax.TransferableOutput) bool {
 		return ok && out1.Asset == out2.Asset && outEq1.Equal(out2.Out)
 	})
 }
+
+func getBaseFee(s state.Chain, cfg *config.Config) (uint64, error) {
+	fee, err := s.GetBaseFee()
+	switch err {
+	case database.ErrNotFound:
+		return cfg.TxFee, nil
+	case nil:
+		return fee, nil
+	}
+	return 0, err
```

Comment:
this method feels a bit weird. i think it should panic here instead of returning 0, err.
Furthermore, IDE suggest alternative switch to leverage errors.Is in the case that ErrNotFound gets wrapped around:
```go
	switch {
	case errors.Is(err, database.ErrNotFound):
		return cfg.TxFee, nil
	case err == nil:
		return fee, nil
	}
```

Reply:
It absolutely shouldn't panic. Error here is just like any other normal error in other GetFromState methods - return zero-value and error.

Reply:
Regarding error.Is, dunno
We expect exactly this single not-found error. And in all other stat-acces method calls we're checking for single-lvl error. If we're going to change it, I'd rather to see it as consistent change across all places for consistency